### PR TITLE
Add helper functions 

### PR DESCRIPTION
**Pull request by ChhuongLe**
Added functions:
1. getAverageRatingById
  1a. Returns the average rating for a specified product
2. getAtelierReview 
    2a. Gets reviews for a specified item
3. getProductsById
   3a. Get product data for a specified item
4. getRelatedItems
    4a. Get an array of related product ids

### CODE REVIEW

File: helpers/getRelatedItems.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      // eslint-disable-next-line no-console
+      console.log(res.data);
+    })
+    .catch((err) => {
+      // eslint-disable-next-line no-console
+      console.log('Error: ', err);
+    });
+};
+
+getRelatedItems(25170);
```
Comment (katehyerinjeon) [inline helpers/getRelatedItems.js:22]: Comment this out?

**Comment by katehyerinjeon** [comment]
What if we organize files by API types (products, reviews, cart, interactions) instead of each function? So we change the filename _getAtelierProducts.js_ to _productsAPI.js_ and the function name _getAtelierProducts_ to _getAllproducts_. Then put functions _getAverageRatingById_, _getProductsById_, _getRelatedItems_ into _productsAPI.js_ and _getAtelierReview_ into _reviewsAPI.js_.

Very nice job on the code. Nothing to review :)

**Review by stl026 — COMMENTED** [review]
Where are the tests for each of these functions Mr. Chhuong

File: helpers/getRelatedItems.js (hunk 1 above)
Comment (stl026) [reply helpers/getRelatedItems.js:22]: Agreed

**Review by stl026 — APPROVED** [review]
Looks good to me if you've tested all of them. Merge them whenever

**Pull request closed without merging**